packages/version/README.md

```diff
@@ -444,6 +445,31 @@ lerna version prepatch --preid next
 When run with this flag, `lerna version` will increment `premajor`, `preminor`, `prepatch`, or `prerelease` semver
 bumps using the specified [prerelease identifier](http://semver.org/#spec-item-9).
 
+### `--package-lockfile-only`
```

Comment:
I'd also mention that the workspace: protocol is recommended when using pnpm because it makes it impossible to fetch packages from the registry accidentally.